docker/package/tezos_setup_wizard.py

```diff
@@ -21,13 +21,13 @@
 
 networks = {
     "mainnet": "Main Tezos network",
-    "hangzhounet": "Test network using version 011 of Tezos protocol (Hangzhou)",
     "ithacanet": "Test network using version 012 of Tezos protocol (Ithaca2)",
+    "jakartanet": "Test network using version 013 of Tezos protocol (PtJakart2)",
```

Comment:
```suggestion
    "jakartanet": "Test network using version 013 of Tezos protocol (Jakarta2)",
```

Nitpick, just for consistency